src/main/java/org/openstreetmap/atlas/geography/atlas/change/FeatureChange.java

```diff
@@ -403,7 +403,8 @@ public ItemType getItemType()
     @Override
     public int hashCode()
     {
-        return Objects.hash(this.changeType, this.afterView);
+        return Objects.hash(this.changeType, this.afterView,
+                this.afterView instanceof Relation ? ((Relation) this.afterView).members() : null);
```

Comment:
It is a great find, however this would also happen for node's in/out edge lists, no? It would be nice to account for that case here too, if possible!

Reply:
Yes good call. Also included a catch-all for any other Atlas Entity afterView that hashes that entity's tags. Hopefully this will help collisions that could result from hashing 2 FC ADDS on an entity with different tag updates.

This is honestly a pretty arbitrary way to do it since the ADD could be a geometry change, or in the case of Nodes, a tag change, and so on. But performance already seems to have improved in cases where we have large hash collections, so we might get away with just picking some property and sticking with it !